README.md

```diff
@@ -78,7 +78,8 @@ CLI options are passed when invoking the `ai` commend:
 - `--interactive` (or `-i`): start an interactive session where user can ask follow-up questions
 - `--provider [name]` (or `-p [name]`): select an inference provider to use: `openai` or `perplexity` (alias `pplx`). You should have relevant API key in your `~/.airc` file.
 - `--model [name]` (or `-m [name]`): select a model to use. This should be a model available for selected provider.
-- `--verbose` (or `-V`): enable verbose mode
+- `--verbose` (or `-V`): enable verbose
+- `--color`: Forces color output (even if stdout is not a terminal). Use `--no-color` to disable colors.
```

Comment:
wouldn't it make sense to make "--no-color" as an option, since --color is by default?

Reply:
default is autodetect, `--color` is force colors even if output is not a terminal, `--no-color` is disable colors even if output is termianl. This is the same as Jest option of the same name.